src/frost.rs

```diff
@@ -535,7 +535,8 @@ fn gen_lagrange_coeff(
         return Err("Duplicate shares provided");
     }
 
-    // TODO: handle this unwrap better like other CtOption's
+    // We know `den` is not zero by the return above.
+    // So we can unwrap() the denominator inverse safely here.
     let lagrange_coeff = num * den.invert().unwrap();
```

Comment:
Use `expect` with a message, rather than explaining it in a comment.

That way, if the expectation ever breaks, you get a useful error message.

Reply:
`CtOption` don't have an `expect` method.

https://docs.rs/subtle/2.1.1/subtle/struct.CtOption.html